packages/system/src/Report/api/hooks.ts

```diff
@@ -29,20 +28,15 @@ const useReportApi = () => {
 
 export const useReports = (category?: ReportCategory) => {
   const api = useReportApi();
-  const initialFilterBy = category
-    ? { category: { equalTo: category } }
-    : undefined;
-  const initialListParameters = {
-    initialSortBy: { key: 'name' },
-    initialFilterBy,
+  const filterBy = category ? { category: { equalTo: category } } : null;
+  const queryParams = {
+    filterBy,
+    sortBy: { key: 'name', isDesc: false, direction: 'asc' as 'asc' | 'desc' },
```

Comment:
thought to just hard code these, as the params are not changed by the UI / user. we are just returning a lookup list. If we add in controls, we can support params again.